tool/tsh/common/mfa.go

```diff
@@ -31,7 +31,7 @@ import (
 	"strings"
 	"time"
 
-	"golang.org/x/exp/slices"
+	"slices"
```

Comment:
Weird that this one hasn't been re-arranged back into the first block of imports.